src/main/java/org/opensearch/knn/common/KNNFaissUtil.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.knn.common;
+
+import java.util.Map;
+
+import static org.opensearch.knn.index.util.Faiss.FAISS_BINARY_INDEX_DESCRIPTION_PREFIX;
+
+public class KNNFaissUtil {
```

Comment:
this is not a faiss util. You can always pass the KNNEngine and the parameters map to know if the index is a binary index or not, this will avoid calling this utility as just faiss utility.